odoo/odoo
# [FIX] crm: lead.scoring.frequency must be accessible by admin

How to reproduce the bug ?

- install the CRM and sales modules
- remove the user from the all the sales security group
- go back to the dashboard and click on settings

What is the bug ?

The res.config.settings model of the CRM module has a field referencing
the crm.lead.scoring.frequency model. This model can only be accessed by
the members of the sale security groups. If an admin is not part of the
sales group, he will get an error when he wants to access the settings
while he will not see the sales or the crm settings tab.

opw-2755177

Signed-off-by: Adrien Minet <admi@odoo.com>

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr
